Makefile

```diff
@@ -53,6 +53,11 @@ TIMESTAMP:=$${__TIMESTAMP:-$(shell /bin/date -u "+%Y%m%d%H%MZ")}
 SHORT_SHA=`git rev-parse --short HEAD`
 BRANCH=`git rev-parse --abbrev-ref HEAD | tr -- '-/' '_'`
 
+# In case the top commit Depends-On some framework patch that is not merged yet - override framework master with the
+# proper version
+DEPENDS_ON_PR=$(shell git show HEAD -q | grep Depends-On | grep -Eo '[[:digit:]]*')
+DEPENDS_ON_PR_IS_OPEN=$(shell curl "https://api.github.com/repos/oamg/leapp/pulls/${DEPENDS_ON_PR}" | python -c "import json,sys; print (json.load(sys.stdin)['closed_at'] is None)")
```

Comment:
@fernflower just quick view. I think these vars should have more specific name. like: REQ_LEAPP_PR - depends on PR is not intuitive. As well, do not use please `$(...)` syntax in Makefile for subshell. It could be missleading for people that are not familiar with Makefile. Use backticks instead.

Reply:
Hmm @pirat89 but backticks don't work for my specific use case - what I need is not an as-is command substitution behavior the backticks give me, but the result of the shell command, and afaik this can only be achieved by a subshell command execution the way I have it now. 
Besides we already have shell command invocation in Makefile, so I would not call it me bending the rules https://github.com/oamg/leapp-repository/blob/master/Makefile#L52

Could you give me a working example with backticks for my specific case?

Reply:
I'm OK with using `$(...)` in the Makefile:
1. Argument that people unfamiliar with Makefile syntax is void - people unfamiliar with `${SUBSTITUTE_ANYTHING}` syntax should learn it first and then they can work with `${SUBSTITUTE_ANYTHING}`
2. The syntax is aleady used in multiple places in this Makefile, the same holds for `leapp`'s Makefile
3. I'm hugely biased against backticks, they should have died really long ago, but that's no argument...